sdk/notificationhubs/arm-notificationhubs/package.json

```diff
@@ -8,78 +8,65 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@azure/abort-controller": "^1.0.0",
-    "@azure/core-auth": "^1.6.0",
-    "@azure/core-client": "^1.7.0",
-    "@azure/core-lro": "^2.5.4",
-    "@azure/core-paging": "^1.2.0",
-    "@azure/core-rest-pipeline": "^1.14.0",
-    "tslib": "^2.2.0"
+    "@azure/abort-controller": "^2.1.2",
+    "@azure/core-auth": "^1.9.0",
+    "@azure/core-client": "^1.9.2",
+    "@azure/core-lro": "^2.7.2",
+    "@azure/core-paging": "^1.6.2",
+    "@azure/core-rest-pipeline": "^1.18.0",
+    "tslib": "^2.8.1"
   },
   "keywords": [
     "node",
     "azure",
     "typescript",
     "browser",
-    "isomorphic"
+    "isomorphic",
+    "cloud"
   ],
   "license": "MIT",
-  "main": "./dist/index.js",
-  "module": "./dist-esm/src/index.js",
-  "types": "./types/arm-notificationhubs.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
-    "@azure/identity": "^4.0.1",
-    "@types/chai": "^4.2.8",
-    "@types/mocha": "^10.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "^4.5.0",
     "@types/node": "^18.0.0",
-    "chai": "^4.2.0",
+    "@vitest/coverage-istanbul": "^2.1.8",
     "dotenv": "^16.0.0",
-    "mocha": "^11.0.2",
-    "ts-node": "^10.0.0",
-    "typescript": "~5.6.2"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Azure/azure-sdk-for-js.git"
+    "playwright": "^1.49.1",
+    "typescript": "~5.6.2",
+    "vitest": "^2.1.8"
   },
+  "repository": "github:Azure/azure-sdk-for-js",
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "files": [
-    "dist/**/*.js",
-    "dist/**/*.js.map",
-    "dist/**/*.d.ts",
-    "dist/**/*.d.ts.map",
-    "dist-esm/**/*.js",
-    "dist-esm/**/*.js.map",
-    "dist-esm/**/*.d.ts",
-    "dist-esm/**/*.d.ts.map",
-    "src/**/*.ts",
+    "dist/",
     "README.md",
-    "LICENSE",
-    "tsconfig.json",
-    "review/*",
-    "CHANGELOG.md",
-    "types/*"
```

Comment:
I think these two should be kept?

Reply:
The types are already in the `dist` folder so no need to keep the `types`.  Most packages do not ship a changelog and just the README is sufficient.

Reply:
@mpodwysocki our changelog tool need the last review folder and changelog to compare and generate the new changelog